showcase/e2e/.eslintrc.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
This file could be simplified since the rules are already defined in the root file:

```json
{
  "extends": "../.eslintrc.json",
  "ignorePatterns": ["!**/*"],
  "overrides": [
    {
      "files": ["*.ts"],
      "parserOptions": {
        "project": ["tsconfig.e2e.json"],
        "createDefaultProgram": true
      },
      "rules": {
        "@angular-eslint/directive-selector": [
          "error",
          {
            "type": "attribute",
            "prefix": "",
            "style": "camelCase"
          }
        ],
        "@angular-eslint/component-selector": [
          "error",
          {
            "type": "element",
            "prefix": "",
            "style": "kebab-case"
          }
        ]
      }
    }
  ]
}
```